### ISSUE TITLE
Ansible looks in the wrong place for python

### ISSUE THREAD

**Issue by tgiardina** [w3f/polkadot-secure-validator#32]
### Setup
I'm using this tool with GCP & AWS sentries and a Packet validator. 

### Issue
When the executing the Ansible playbook, the GCP & AWS instances throw "/bin/sh: 1: /usr/bin/python: not found" because Python's binary is located at /usr/bin/python3.

### Temporary Solution
I manually SSHed into the servers and moved the binary location.

### Permanent Solution
Ansible needs to confirm location of python before executing /usr/bin/python. I will try to make a pull request with this fix over the next few days.

**Comment by fgimenez** [comment]
Thanks for reporting!

Looks like you are using just the playbooks on a host created by you, if you use the scripts in this repo the hosts created by terraform don't have the problem you describe.

Take a look at https://docs.ansible.com/ansible/latest/reference_appendices/interpreter_discovery.html for running ansible on a host that doesn't have the python interpreter on the common locations, closing this issue now.

**Comment by tgiardina** [comment]
Thanks for the quick responses, Federico!

I am using the repo's scripts (i.e., `yarn sync -c config/main.json` and `yarn clean -c config/main.json`), not just the playbooks. There are also a few other small manual changes that I had to make in order to get the script to work as desired. 

I will look deeper into these problems tomorrow to make sure that they're not configuration mistakes on my end. I will report back soon either way.

**Comment by fgimenez** [comment]
Interesting, the terraform modules in this repo by default create hosts based on ubuntu 18.04, whuch ships a python interpreter in the proper location. You can check the integration test results in the CI build to see how ansible works on this kind of hosts.

Please let us know if you find the cause of your issue, thx!

**Comment by tgiardina** [comment]
The issues were indeed configuration problems. Here they are (with solutions) for posterity:
1) The servers were "unreachable" depending on where I put my ssh key. 
**Cause:**
I wasn't adding the keys via ssh-add.
**Solution:**
I added the keys via ssh-add after creating them with ssh-keygen.
2) Ansible was looking in the wrong place for python
**Cause:**
I had an old version of Ansible (Thanks for the interpreter_discovery link! I immediately realized my mistake when reading through it!) 
**Solution:**
I installed Ansible through pip rather than through apt.

Great job on the package! It definitely saved me a lot of time! I think it would be nice if the README was a bit more expansive, though. I wrote an [extremely verbose walkthrough](https://github.com/tgiardina/secure-validator-walkthrough) for myself, but I think it may be too much for this repo's README. Instead, I'll make a pull request with a just few small additions to the README sometime over the next few days.
